packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -345,14 +346,15 @@ export const mutation = mutationType({
       },
     })
 
-    t.nonNull.field('setProjectPreferences', {
+    // TODO: #23202 hopefully we can stop using this for project data, and use `setPreferences` instead
+    t.nonNull.field('setProjectPreferencesInGlobalCache', {
```

Comment:
Worth marking this as deprecated? I believe there's a `deprecation` field we can define alongside the `description` to provide info on preferred alternative(s)

Reply:
I think I'd like to actually poke at it a bit before being sure it is deprecated. Maybe there's some lifecycle reason that this is used that we can comment as the outcome of #23202.